src/main/java/core/basesyntax/ChangeToCar.java

```diff
@@ -1,9 +1,11 @@
 package core.basesyntax;
 
-/**
- * <p>Имплементируйте необходимый функциональный интерфейс и его метод, который принимает вес авто
- * и возвращает тип авто в зависимости от веса (если weight меньше 1500, то CAR, если больше, то
- * TRUCK). Метод должен принимать Integer и возвращать AutoType</p>
- */
-public class ChangeToCar {
+import java.util.function.Function;
+
+public class ChangeToCar implements Function<Integer, AutoType> {
+
+    @Override
+    public AutoType apply(Integer weight) {
+        return weight < 1500 ? AutoType.CAR : AutoType.TRUCK;
```

Comment:
see checklist or think yourself what might be wrong here

Reply:
need a constant probably